libs/datasets/timeseries.py

```diff
@@ -96,35 +96,19 @@ def latest_values(self, aggregation_level=None) -> pd.DataFrame:
     def get_subset(
         self,
         aggregation_level,
-        on=None,
-        after=None,
-        before=None,
```

Comment:
fwiw I use `on`, `after`, and `before`  a lot when I'm debugging data locally (in an ipython notebook). same with `state`.